ecs-logs-collector.sh

```diff
@@ -443,7 +443,7 @@ get_containers_info()
   if [[ "$?" -eq 0 ]]; then
     mkdir -p ${info_system}/docker
 
-    for i in `docker ps |awk '{print $1}'|grep -v CONTAINER`; do
+    for i in `docker ps -q`; do
```

Comment:
I prefer this, I believe we had this transformed with `awk` for backwards compatibility. I can't find exactly when this was added to `docker ps`, but I think this is fine to move forward with.